### PR TITLE
Feature/api calls

### PR DESCRIPTION
**Pull request by Gabby-Recny**
### What (if any) features are you implementing?
- Fetching API calls for single movies and entire movie library
### What (if anything) did you refactor?
- N/A
### Were there any issues that arose?
- Curious if we need specific error handling for 500 statuses and how to test 500 errors
### Is there anything that you need from your teammate?
- N/A
### Refactor checklist
- [ ] Check variable and parameter names: are they descriptive and unique?
- [ ] Is the code DRY? SRP?
- [ ] Is it easy for others to read your code? (explanatory comments needed?)
- [ ] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [ ] Are commit messages formatted correctly?

### CODE REVIEW

**Review by jasmyn2244 — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import logo from './logo.svg';
 import './App.css';
 import Header from './Header.js';
-import movieData from './dummy-data';
```
Comment (anewb87) [inline src/App.js:5]: Fetch calls are working, no need for this.

File: src/App.js [hunk 2]
```diff
@@ -13,30 +12,71 @@ class App extends React.Component {
       movies: [],
       selectedMovie: {},
       isSelected: false,
+      error: '',
+      errorMsg: ''
```
Comment (anewb87) [inline src/App.js:16]: Went down one path with error handling. Aren't currently using this but might circle back.

File: src/App.js [hunk 3]
```diff
@@ -13,30 +12,71 @@ class App extends React.Component {
...
     };
   }
 
   componentDidMount = () => {
-    this.setState({movies: movieData.movies})
+      fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
```
Comment (anewb87) [inline src/App.js:21]: Hey Robbie- do we need a file specifically for API calls?

File: src/MovieDetails.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ const MovieDetails = (props) => {
       <section>
         <button className='back-to-main-button' onClick={() => props.displayMain()}>Back to Main</button>
         <img className='movie-backdrop' src={props.selectedMovie.backdrop_path} alt={props.selectedMovie.title} />
-        <p className='gradient'></p>
+        // <p className='gradient'></p>
```
Comment (anewb87) [inline src/MovieDetails.js:10]: We love the gradient but realize this p tag is actually causing functionality issues when the screen size chages.

**Review by anewb87 — APPROVED** [review]
Have some questions for instructors-
The major one being how to handle 500 errors. The spec is a little unclear- do we need a message specifically for 500 errors OR is it telling us 'hey, remember to handle 500 errors too' ?

**Pull request merged**